### PR TITLE
DBZ-3953 Mark wal2json plugin as deprecated

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-3953

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -2021,24 +2011,10 @@ It is possible to capture changes in a PostgreSQL database that is running in li
...
-* PostgreSQL 9.6: 9.6.10 and newer
-* PostgreSQL 10: 10.5 and newer
-* PostgreSQL 11: any version
-====
-
 [[postgresql-on-azure]]
 ==== PostgreSQL on Azure
 
-It is possible to use {prodname} with link:https://docs.microsoft.com/azure/postgresql/[Azure Database for PostgreSQL], which has support for the `wal2json` and `pgoutput` plug-ins, both of which are supported by {prodname} as well.
+It is possible to use {prodname} with link:https://docs.microsoft.com/azure/postgresql/[Azure Database for PostgreSQL], which has support for the and `pgoutput` plug-in, which is supported by {prodname} as well.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2017]:
```suggestion
It is possible to use {prodname} with link:https://docs.microsoft.com/azure/postgresql/[Azure Database for PostgreSQL], which has support for the `pgoutput` logical decoding plug-in, which is supported by {prodname}.
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 2]
```diff
@@ -2066,19 +2042,13 @@ While using the `pgoutput` plug-in, it is recommended that you configure `filter
...
 
-The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to encode the changes in either link:https://github.com/google/protobuf[Protobuf format] or link:http://www.json.org/[JSON] format.
-See the documentation for your chosen plug-in to learn more about the plug-in's requirements, limitations, and how to compile it.
+The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to encode the changes in either link:https://github.com/google/protobuf[Protobuf format] or link:https://github.com/postgres/postgres/blob/master/src/backend/replication/pgoutput/pgoutput.c[pgoutput] format.
+`pgoutput` plugin comes out-of-the-box with PostgreSQL server.
+For other plug-in see the documentation for your chosen plug-in to learn more about the plug-in's requirements, limitations, and how to compile it.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2049]: That sounds a bit odd, given that the "other plug-in" is maintained by ourselves.

**Review by gunnarmorling — COMMENTED** [review]
LGTM overall; a few minor remarks/suggestions inline.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc (hunk 2 above)
Comment (jpechane) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc]: @gunnarmorling Rephrased

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 3]
```diff
@@ -58,7 +58,7 @@ The PostgreSQL connector contains two main parts that work together to read and
 ifdef::community[]
 * A logical decoding output plug-in. You might need to install the output plug-in that you choose to use. You must configure a replication slot that uses your chosen output plug-in before running the PostgreSQL server. The plug-in can be one of the following:
 ** link:https://github.com/debezium/postgres-decoderbufs[`decoderbufs`] is based on Protobuf and maintained by the {prodname} community.
-** link:https://github.com/eulerto/wal2json[`wal2json`] is based on JSON and maintained by the wal2json community.
+** link:https://github.com/eulerto/wal2json[`wal2json`] is based on JSON and maintained by the wal2json community (deprecated, scheduled for removal).
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:61]:
```suggestion
** link:https://github.com/eulerto/wal2json[`wal2json`] is based on JSON and maintained by the wal2json community (deprecated, scheduled for removal in Debezium 2.0).
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 4]
```diff
@@ -2066,19 +2042,11 @@ While using the `pgoutput` plug-in, it is recommended that you configure `filter
...
 
-The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to encode the changes in either link:https://github.com/google/protobuf[Protobuf format] or link:http://www.json.org/[JSON] format.
-See the documentation for your chosen plug-in to learn more about the plug-in's requirements, limitations, and how to compile it.
-
-* link:https://github.com/debezium/postgres-decoderbufs/blob/main/README.md[`protobuf`]
-* link:https://github.com/eulerto/wal2json/blob/master/README.md[`wal2json`]
+The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to encode the changes in either link:https://github.com/google/protobuf[Protobuf format] or link:https://github.com/postgres/postgres/blob/master/src/backend/replication/pgoutput/pgoutput.c[pgoutput] format.
+`pgoutput` plugin comes out-of-the-box with PostgreSQL server.
+For more details Protobuf plug-in see the link:https://github.com/debezium/postgres-decoderbufs/blob/main/README.md[`documentation`] to learn more about the plug-in's requirements, limitations, and how to compile it.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2049]:
```suggestion
The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to receive change events from the database in either the link:https://github.com/google/protobuf[Protobuf format] or the link:https://github.com/postgres/postgres/blob/master/src/backend/replication/pgoutput/pgoutput.c[pgoutput] format.
The `pgoutput` plugin comes out-of-the-box with the PostgreSQL database.
For more details on using Protobuf via the `decoderbufs` plug-in, see the plug-in link:https://github.com/debezium/postgres-decoderbufs/blob/main/README.md[`documentation`] which discusses its requirements, limitations, and how to compile it.
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 5]
```diff
@@ -2115,9 +2075,9 @@ If you are using a {link-prefix}:{link-postgresql-connector}#postgresql-output-p
 [source,properties]
 ----
 # MODULES
-shared_preload_libraries = 'decoderbufs,wal2json' // <1>
+shared_preload_libraries = 'decoderbufs' // <1>
 ----
-<1> Instructs the server to load the `decoderbufs` and `wal2json` logical decoding plug-ins at startup. The names of the plug-ins are set in the link:https://github.com/debezium/postgres-decoderbufs/blob/v0.3.0/Makefile[`Protobuf`] and link:https://github.com/eulerto/wal2json/blob/master/Makefile[`wal2json`] make files.
+<1> Instructs the server to load the `decoderbufs` logical decoding plug-ins at startup. The names of the plug-ins are set in the link:https://github.com/debezium/postgres-decoderbufs/blob/v0.3.0/Makefile[`Protobuf`] make file.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:2080]:
```suggestion
<1> Instructs the server to load the `decoderbufs` logical decoding plug-ins at startup (the name of the plug-in is set in the link:https://github.com/debezium/postgres-decoderbufs/blob/v{debezium-version}/Makefile[`Protobuf`] make file).
```

**Review by gunnarmorling — APPROVED** [review]
Ok, going to amend this with some more tweaks and will merge. Please take a look at them and let me know if you see any issues with them. Thx!

**Pull request merged**